src/components/templates/ConversationSpace/ConversationSpace.scss

```diff
@@ -315,3 +320,22 @@
     }
   }
 }
+
+.conversation-space-background {
```

Comment:
this should go to ContainerWithBackground, i believe.

Reply:
fixed b396ea2